tests/testthat/test-summarize_weighted_production.R

```diff
@@ -420,3 +420,24 @@ test_that("with known input outputs as expected", {
     summarize_weighted_percent_change(name_ald, use_credit_limit = TRUE)
   expect_equal(out2$weighted_percent_change, c(0, 0, 40, 180))
 })
+
+test_that("with duplicated currencies errors with informative message (#137)", {
```

Comment:
Is "duplicated" the best word? The error class suggests "different currencies" is better than "duplicated currencies".

Reply:
yes you are right